apps/web/modules/settings/my-account/components/CompanyEmailOrganizationBanner.tsx

```diff
@@ -0,0 +1,84 @@
+"use client";
+
+import { useRouter } from "next/navigation";
+
+import { useFlagMap } from "@calcom/features/flags/context/provider";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { Button } from "@calcom/ui/components/button";
+
+import { MailIcon } from "./MailIcon";
+
+type CompanyEmailOrganizationBannerProps = {
+  onDismissAction: () => void;
+};
+
+export const CompanyEmailOrganizationBanner = ({ onDismissAction }: CompanyEmailOrganizationBannerProps) => {
+  const { t } = useLocale();
+  const router = useRouter();
+  const flags = useFlagMap();
+
+  const handleLearnMore = () => {
+    const redirectPath = flags["onboarding-v3"]
```

Comment:
It'd be really nice to centralize all the redirect paths regarding the onboarding. But I guess once we roll out v3 stably, will we remove all the v3 code and paths?